WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -125,7 +125,6 @@ decisions = {
 					NOT = { culture = ROOT }
 				}
 			}
-			holy_order = no
```

Comment:
Given the nature of holy orders the characters culture will switch with time. It makes sense to have holy order = no out of historical reasons.